catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/IngestCommand.java

```diff
@@ -369,6 +471,9 @@ private Metacard readMetacard(File file) throws IngestException {
             }
         }
 
+        if (!ignoreContent) {
+            createContentItem(result);
```

Comment:
We can't do this here. What you're doing is creating the content item before we've even determined that the system or the user is allowed to ingest the metacard. This is bypassing all of the security controls in the catalog framework. You need to only go through the catalog framework with the file stream and allow it to generate the metacard and attempt to write to the content store.

👎  until this is resolved.